jadn-v1.1.md

```diff
@@ -154,46 +159,64 @@ For complete copyright information please see the Notices section in the Appendi
-an information model is to model objects at a conceptual level, independent of specific implementations
-or protocols used to transport the data.
-[RFC 8477](#rfc8477), "IoT Semantic Interoperability Workshop 2016", describes a lack of consistency across
-Standards Developing Organizations in defining application layer data, attributing it to the lack of an
-encoding-independent standardization of the information represented by that data.
-This document defines an information modeling language intended to address that gap. JADN is a
-[formal description technique](#fdt) that combines type constraints from the Unified Modeling Language
-[UML](#uml) with data abstraction based on information theory and structural organization using results
-from graph theory.
-
-As shown in Figure 1, industry has multiple, often conflicting definitions of data modeling terms,
-including the term "[Information Engineering](#ie)", which at one time referred to
-[data modeling](#datamod) but is now more closely aligned with information theory and machine learning.
-
-* Ackoff's [Knowlege Hierarchy](#diek) defines data as "symbols that are properties of observables"
-  and informally calls information "descriptions inferred from data".
-* UML defines DataTypes (simple classifiers where instances are distinguished only by value) and
-  Classes (structured classifiers where instances have behavior, inheritance, roles, and other
-  complex characteristics).
-* Traditional data modeling defines conceptual, logical and physical data models without considering
-  information at all.
-* Information modeling formalizes the relationship between information and data, defining a
-  technology-agnostic information layer that lies between the logical data model and
-  multiple technology-specific physical data models.
-
-![Information Engineering](images/info-engineering.jpg)
-
-###### Figure 1: Information Engineering Terminology
-
-UML class models and diagrams are commonly referred to as "Data Models", but they model knowledge
-of real-world entities using classes. In contrast, information models model data itself using datatypes.
-A practical distinction is that class models are undirected graphs with an unlimited variety of
-classes and semantic relationships, while information models are directed graphs with a small predefined
-set of base datatypes and only two kinds of relationship: "contain" and "reference".
-Designing an information model from a class/logical model is largely a matter of assigning the kind and
-direction of each relationship, establishing identifiers for all referenceable datatypes, and
-selecting the kind of each datatype from among the base types defined by an information modeling
-language. Converting an information model to a data model means applying serialization rules
-for each base type that produce physical data in the desired format.
+> *An information model is a representation of concepts, relationships, constraints, rules,
```

Comment:
You've got 3 quotations in here; Y Tina Lee, UML and RDF, but they aren't formatted the same. Suggest normalizing the format as a note (>) in italics w/o quotation marks.